lib/fathead/mdn_css/fetch.pl

```diff
@@ -57,12 +57,15 @@
                         spurt $tx->res->body, catfile 'download',
                           "$file_number.html";
                         ++$file_number;
-                        
-                        #if the keyword URL redirects to another URL, write the keywords to a file
-                        if($url ne $tx->req->url) {
-                            my $keyword  = substr ($url, rindex($url, "/") + 1);
-                            my $redirect = substr ($tx->req->url, rindex($tx->req->url, "/") + 1);
-                            $redirect_map{$redirect} = $keyword;
+
+      #if the keyword URL redirects to another URL, write the keywords to a file
```

Comment:
I will look into it. Could be caused by Perltidy as I have set it such that it runs after each save.